site/en/docs/search_reranking_using_embeddings.ipynb

```diff
@@ -31,19 +31,6 @@
         "# limitations under the License."
```

Comment:
<p>This examples link needs to be absolute.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>Done.</p>